pkg/tracking/store/sql/experiments_search_builder.go

```diff
@@ -258,6 +258,7 @@ func applyExperimentsFilter(database, query *gorm.DB, filter string) (*gorm.DB,
 	return query, nil
 }
 
+//nolint:gosec // disable G115
```

Comment:
Could you create a follow-up issue for these violations?
(Did you try and fix them?)

Reply:
Created, I checked around for those violations and opened this issue https://github.com/mlflow/mlflow-go/issues/73